test/featureflags.go

```diff
@@ -32,3 +32,20 @@ func requireAnyGate(gates map[string]string) func(context.Context, *testing.T, *
 		t.Skipf("No feature flag matching %s", strings.Join(pairs, " or "))
 	}
 }
+
+// getEmbeddedStatusValue gets the current value for the "embedded-status" feature flag.
+// If the flag is not set, it returns the default value. This is used in tests which look
```

Comment:
nit: better not to include in a function's docstring details of how the function is used.

Reply:
Okiedokie, cleaning it up!